package.json

```diff
@@ -48,6 +48,7 @@
     "formio-utils": "^0.2.4",
     "formiojs": "^1.5.4",
     "jquery": "^2.2.4",
+    "lodash": "^4.17.4",
```

Comment:
if you dont want to include the whole lodash lib, I made a factory for select methods: https://github.com/formio/ngFormio/blob/master/src/factories/Lodash.js